bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/NvsBeanValidator.java

```diff
@@ -144,7 +145,7 @@ private String validateEncoding(String encoding, String type)
 		if (!List.of(NvsTableDataService.getEncodings(type)).contains(encoding))
 		{
 			return String.format(Messages.NvsValidation_EncodingValidationErr_1, type,
-					NvsTableDataService.getEncodings(type), encoding);
+					String.join(",", NvsTableDataService.getEncodings(type)), encoding); //$NON-NLS-1$
```

Comment:
getEncodings returning an array, so we need to use String.join to format the error message